src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.scss

```diff
@@ -79,17 +73,31 @@
     height: 24px;
     margin-left: 8px;
     position: relative;
-    width: 30px;
+    width: 26px;
   }
 
   .trackHeightBtn {
-    height: 33px;
-    position: relative;
+    height: 12px;
     width: 50px;
   }
 
+  .trackHighlightBtn {
+    height: 18px;
+    width: 18px;
+  }
+
+  .trackConfigBtn {
+    height: 18px;
+    width: 18px;
+  }
+
   .trackMoveBtn {
-    height: 26px;
-    width: 26px;
+    height: 28px;
+    width: 14px;
+  }
+
+  .trackLockBtn {
+    height: 18px;
+    width: 18px;
```

Comment:
Since `trackHighlightBtn`, `trackConfigBtn` and `trackLockBtn` have identical styles, let's write them together:

```
.trackHighlightBtn, . trackConfigBtn, .trackLockBtn {
    height: 18px;
    width: 18px;
}
```